src/components/transactions/eip5792/sendCalls.js

```diff
@@ -0,0 +1,254 @@
+import globalContext from '../../..';
+
+const DEFAULT_CALLS = [
+  {
+    to: '0x0c54FcCd2e384b4BB6f2E405Bf5Cbc15a017AaFb',
+    data: '0x654365436543',
+    value: '0x1234123412341234',
+  },
+  {
+    to: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+    data: '0x789078907890',
+    value: '0x4321432143214321',
```

Comment:
just a note, maybe in a future PR we could add contract interactions in the batch too (like approve, transfer etc etc).
Summary of combinations that seem important:
- batch sends all tx benign
- batch send with malicious tx
- batch sends + contract interactions all benign
- batch sends + contract interactions with malicious
- batch send/contract with a failing tx

I can support on adding those if this make sense to you too :pray:

Reply:
A contract interaction would be great, but harder to default as we'd have to use known addresses on each chain.

The `Edit Request` would let you test any combination manually.

I think default buttons for malicious and failing make sense in future, but the malicious one for example can't be tested yet until it's on Sepolia for example or a chain that Blockaid supports.

Reply:
the Edit Request is :fire: